compiler-rt/lib/scudo/standalone/checksum.cpp

```diff
@@ -19,6 +19,8 @@
 #else
 #include <sys/auxv.h>
 #endif
+#elif defined(__loongarch_lp64)
```

Comment:
Hmm, why not just __loongarch__?  Even if some LA32 implementations do not support CRC32 we can detect it with HWCAP anyway.

Reply:
I was following the manual word-for-word. But I think your suggestion is reasonable and I'll change it later.

Reply:
The manual basically says "LA32 SHALL implement the instructions in table 2-1", but there is no clause saying "LA32 SHALL NOT implement other instructions".

Reply:
Done.